README.md

```diff
@@ -53,6 +53,36 @@ To run all checks execute the following command:
 ./gradlew check
 ```
 
+### Docker
+
+To run API in docker container execute the following:
+
+```bash
+./bin/run-in-docker.sh
+```
+
+If you already ran this command before or stepped through manually, just play around with docker commands and follow the instructions
+
+```bash
+docker-compose up
+```
+
+Script includes bare minimum environment variables necessary to start (no successfully!) database and api instances. Whenever any variable is changed or any other script regarding docker image/container build, the suggested way to ensure all is cleaned up properly is by this command:
```

Comment:
>to start (not successfully!) database and api instances

Let's make this script run the service successfully, otherwise it's useless imho